services/frontend/src/components/mentorshipCard/MentorshipView.jsx

```diff
@@ -2,34 +2,31 @@ import PropTypes from "prop-types";
 import { Tag, Empty, Row, Col } from "antd";
 import { TagTwoTone } from "@ant-design/icons";
 import { FormattedMessage } from "react-intl";
+import { Fragment } from "react";
```

Comment:
please put this at the top (just a personal preference since react is the "main" package) let me know what you think